src/components/InfoPanel.component.js

```diff
@@ -1,8 +1,42 @@
 import React, { Component } from 'react'
+import { Image } from 'react-bootstrap'
+import './../styles/App.css'
 
 class InfoPanel extends Component {
+  constructor(props) {
+    super(props)
+  }
+
   render() {
-    return <div>InfoPanel</div>
+    const selectedEvent = this.props.selectedEvent
+    const img_url = selectedEvent.image
```

Comment:
This variable isn't being used anywhere, so you should remove it. Also in javascript, it is conventional to use camelCase